modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase

```diff
@@ -112,6 +112,49 @@ definition {
 			objectName = "CustomObject184");
 	}
 
+	@description = "LPS-161893 - Verify that it's possible to export an object entry with aggregation field"
+	@priority = "4"
+	test CanExportObjectWithAggregationField {
+		task ("Create Custom Object and Custom Relationship itself (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161893",
+				objectName = "CustomObject161893",
+				pluralLabelName = "Custom Objects 161893");
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObject161893",
+				objectName_2 = "CustomObject161893",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject161893");
+		}
+
+		task ("When go to Custom Object > Fields tab and add Custom Aggregation Field (via UI)") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 161893");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				field = "ID",
+				fieldLabel = "Custom Aggregation",
+				fieldType = "Aggregation",
+				function = "Max",
+				relationship = "Relationship");
+		}
+
+		task ("Then assert the object with aggregation field can be exported") {
```

Comment:
Then: The custom object with a self-relationship and Aggregation field can be exported